### DISCUSSION TITLE
Possible to start with an empty checkpoint?

### DISCUSSION THREAD

**Discussion by mostlyhuman** [d8ahazard/sd_dreambooth_extension#597] [Q&A]
If I wanted to start from the ground up and not use one of the existing training sets is this possible? If so, how would I do it? Cheers!

**Comment by cerega66** [comment]
Do you want to train your model right from zero?

**Reply by ghost** [reply]
That would be an interesting experiment, for example if you want to create an abomination out of all your LORAs :D

Im interested if there is an all zero weights checkpoint available too
